### PR TITLE
changed snake colors and empty space

### PR DESCRIPTION
**Pull request by jadowdl**

### CODE REVIEW

File: Game.js [hunk 1]
```diff
@@ -43,40 +43,40 @@ export default class Game extends React.Component {
   //   }
   // }
 
-  render() {	
+  render() {
     // Determine the size (in pixels) of a single Box.
     var {width, height} = Dimensions.get('window')
-    width -= 100;
-    height -= 100;
+    width -= 50;
```
Comment (westonz7042) [inline Game.js]: wtf happened

File: Game.js [hunk 2]
```diff
@@ -43,40 +43,40 @@ export default class Game extends React.Component {
   //   }
   // }
 
-  render() {	
+  render() {
     // Determine the size (in pixels) of a single Box.
     var {width, height} = Dimensions.get('window')
-    width -= 100;
```
Comment (Imperii) [inline Game.js]: Why so large...

Comment (dragonjws) [inline Game.js]: Why did they remove 100 and add 50?

Comment (Imperii) [reply Game.js]: Stop cyber-bullying others online. Not very nice Justin Shao. Even though you don't understand does not give you the right to bully others.

File: Model.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
-const GRID_WIDTH = 10;
-const GRID_HEIGHT = 10;
+const GRID_WIDTH = 50;
```
Comment (itblursby) [inline Model.js]: 50x50 seems a bit big for snake

File: Game.js (hunk 2 above)
Comment (Captainmolt) [reply Game.js]: cuz 100 is a dumb number

Comment (HippoShark) [reply Game.js]: comment

Comment (dragonjws) [reply Game.js]: what is a smart number?

Comment (westonz7042) [reply Game.js]: ur mom dumb number

File: Game.js [hunk 4]
```diff
@@ -8,7 +8,7 @@ import Box from './Box';
 import Interface from './Interface';
 
 export default class Game extends React.Component {
-  
+
```
Comment (Imperii) [inline Game.js]: Why is there a +

**Pull request merged**